### DISCUSSION TITLE
Hello, help me please start it up :)

### DISCUSSION THREAD

**Discussion by ysruk** [snipem/gt7dashboard#25] [Q&A]
I dont understand what i need to do next, after installing run.command file. It's something with ip adress.:D
<img width="565" alt="Снимок экрана 2024-02-21 в 06 21 44" src="https://github.com/snipem/gt7dashboard/assets/50845878/bbe685b8-8e6d-42c1-aa18-39a083e61786">

**Comment by snipem** [comment]
Replace `GT7_PLAYSTATION_1P=<192.168.0.9>` with `GT7_PLAYSTATION_1P=192.168.0.9`. The brackets are place holders.

**Reply by 036Preme** [reply]
<img width="562" alt="Screenshot 2024-08-09 at 12 05 09" src="https://github.com/user-attachments/assets/2fc962d5-1654-4666-a14d-5e7de54abcac">
@snipem 

I definitely got turned around (1st time ever really using run cmd) all help is appreciated. i somehow am not connected to my playstation but some random port

**Reply by snipem** [reply]
You might have some other instance of gt7dashboard running
